src/types/catalog.d.ts

```diff
@@ -265,6 +265,7 @@ export interface RetrieveSearchabilitiesParameters {
 
 export interface PatchSearchabilitiesParameters {
   searchabilities: SearchabilityConfiguration[],
```

Comment:
Not really a part of your pr, but can we change the end of line here from a comma to semicolon for consistency 😅